src/panels/SessionRecording/SessionRecMenuButton.tsx

```diff
@@ -54,7 +54,7 @@ export function SessionRecMenuButton({ onClick }: SessionRecMenuButtonProps) {
         );
       case RecordingState.paused:
         return (
-          <Group gap={'xs'}>
+          <Group gap={0}>
```

Comment:
Why is this gap changed?  Seems like a mistake in the commit? 

Especially since it leads to a different gap compared to the Pause button below

If so,